### ISSUE TITLE
Satcat / TLE entries virtually all at z > 0

### ISSUE THREAD

**Issue by gomezzz** [schuhmaj/nasa-breakup-model-cpp#37]
We are still seeing some problems reading the TLEs from satcat. It could be this is in the data or it is a bug in the reader. We are double checking on our end, maybe you could also have a brief look?

The problem we see is that there is a very small number of satellites with negative Z coordinates. There are about ~250 negative z coordinate ones, but that seems very low. We'd expect approximately equal amount > and < z=0.

#### LEO
![leo_1](https://user-images.githubusercontent.com/2656700/127897602-24b2a68b-b406-4ad9-9651-c1e7c462f5a6.png)
![leo_2](https://user-images.githubusercontent.com/2656700/127897605-347e8214-a869-4326-a276-c4efb508a740.png)

### GEO
![geo_1](https://user-images.githubusercontent.com/2656700/127897597-cef551d5-6049-47a7-b6d6-b2652aca97af.png)
![geo_2](https://user-images.githubusercontent.com/2656700/127897600-d4975428-42b2-4cc6-a0c4-a0caddaa95ff.png)

**Comment by schuhmaj** [comment]
[2021-08-02 18:37:29.172] [info] Count 1765 with z < 0
[2021-08-02 18:37:29.173] [info] Count 11790 with y < 0
[2021-08-02 18:37:29.173] [info] Count 9289 with x < 0
[2021-08-02 18:37:29.173] [info] From 21244 objects

Thats the result I get from the TLESatcatReader, with the satcat.csv from the repository/ NORAD and the TLE file from last week
Definitely far less negative z coordinates, but if this a bug or just the data. No idea.
On the other hand, the epoch of the TLE sets is partly different in this set, some data is from day 207, 206, 208 and from various times. So, we do not have an exact cut from day X at XX:XX:XX.

The TLEReader does not read in the epoch, currently, because it is not required for the simulation.

My wobbly guess:
Positive Z is northern hemisphere. Given the fact that Space Track is USA, it might be the case that the radar data is produced when a satellite fly's over the northern hemisphere. Furthermore, if this is (still) accurate (https://de.wikipedia.org/wiki/Space_Detection_and_Tracking_System), those stations are mainly in the north.

This is a wobbly theory, but maybe?

**Comment by schuhmaj** [comment]
Oh, 👏 beautiful data.

The fact that satellites in southern LEO seem to be missing would support the "northern hemisphere better radar theory". On the other hand, GEO south --> not blocked by radar???

Are all satellites plotted for the exact same epoch?
If yes, the TLEReader probably has a bug, if not my theory strikes back 👽


_Hmm, otherwise, if we assume that the earth is flat..._

**Comment by gomezzz** [comment]
> Are all satellites plotted for the exact same epoch?

This was indeed the error. It works now :)
